src/blockCache/sqlite.test.ts

```diff
@@ -0,0 +1,86 @@
+import { createSqliteBlockCache } from "./sqlite.js";
```

Comment:
One thing probably worth keeping in mind when using a SQL database is that, because of the consistency requirements, it might enforce sequential writing even for data with no dependencies, and thus be a bottleneck to a degree, compared to a non-relational store.

That said, doing queries with ordering and comparisons would be trickier in non-relational stores.